### ISSUE TITLE
Session Status Missing Feature "Canceled Session" and Missing Option to Revert from "Confirmed" back to "Accepted" (with different colors)

### ISSUE THREAD

**Issue by mariobehling** [fossasia/open-event-frontend#4522]
Currently sessions can have the status accepted, confirmed or rejected. Also when they are submitted they have a dedicated status or a status session rejected. 

However, organizers could mistakenly mark a session "confirmed". In that case they are not able to revert a session to "accepted".

Another thing that happens is, that speakers cancel a session for some reason. In that case organizers currently have to mark a session as "Rejected" in order to get it out of the public sessions list. Please change the feature as follows:

1. Enable status changes from confirmed to accepted or any other status
2. Propose and implement dedicated colors for each status
3. Add the status "Canceled" for sessions.
4. Speakers have the option to "withdraw" a session. This session will not show up in the list of organizers anymore. Change this in the way that "withdrawn sessions" show up with the status "withdrawn".

**Comment by iamareebjamal** [comment]
Will the cancelled session be visible to users in session list/detail view?

**Comment by mariobehling** [comment]
> Will the cancelled session be visible to users in session list/detail view?

Yes, I think the status can also be visible in the user view. At the moment I would not know why not. However the only action users can do after the CfS closes is "withdraw". They cannot change to another status apparently.

I am still unclear if we should also add the status "Draft". A draft status talk would be a talk that is not yet submitted (before the "pending status"). However, if we have this status it makes it more complicated at the moment probably. Cause a draft should not be allowed to be accepted. The user needs to submit the talk actively, I think.

**Comment by iamareebjamal** [comment]
Draft status is already there but don't know if it is implemented correctly

**Comment by iamareebjamal** [comment]
State and Colors:

Draft: 
![image](https://user-images.githubusercontent.com/3874064/88477315-b8a54080-cf5c-11ea-82e2-1d378a5c211b.png)

Accepted:
![image](https://user-images.githubusercontent.com/3874064/88477327-cc50a700-cf5c-11ea-9963-463cfe5171ee.png)

Confirmed:
![image](https://user-images.githubusercontent.com/3874064/88477332-d96d9600-cf5c-11ea-824c-d66fb1d77896.png)

Rejected:
![image](https://user-images.githubusercontent.com/3874064/88477340-ea1e0c00-cf5c-11ea-8fef-c81ba816e304.png)

Withdrawn:
![image](https://user-images.githubusercontent.com/3874064/88477370-05891700-cf5d-11ea-9fac-800430c1a085.png)

Canceled:
![image](https://user-images.githubusercontent.com/3874064/88477377-176aba00-cf5d-11ea-9774-79494b486b89.png)

**Comment by iamareebjamal** [comment]
To be implemented for speakers
